tests/pytests/common.py

```diff
@@ -668,3 +668,35 @@ def verify_shard_init(shard):
 def cmd_assert(env, cmd, res, message=None):
     db_res = env.cmd(*cmd)
     env.assertEqual(db_res, res, message=message)
+
+# fields should be in capital letters
+def getInvertedIndexInitialSize_MB(fields: list[str]) -> float:
+    total_size = 0
+    for field in fields:
+        if field in ['GEO', 'NUMERIC']:
+            block_size = 48
+            initial_block_cap = 6
+            inverted_index_meta_data = 48
```

Comment:
can it be useful somewhere else, and avoid "magic numbers" in function?

Reply:
where is somewhere else? It's in a common file available for any test in this directory
And those are the structs size in our code, how can i get them other than hard coding?
I wrapped it in a function so it could be easily changed in the future if needed